src/main/java/com/example/onlinebookstore/model/User.java

```diff
@@ -30,4 +46,45 @@ public class User {
     @NotBlank
     private String lastName;
     private String shippingAddress;
+    private Boolean isDeleted = false;
```

Comment:
```suggestion
    private boolean isDeleted = false;
```

Also, email should be unique
```
@NotBlank
@Email
private String email;
```